tabs/search/filter.lua

```diff
@@ -9,8 +9,7 @@ local post_filter = {}
 local post_filter_index = 0
 
 function aux.event.AUCTION_HOUSE_LOADED()
-    initialize_class_dropdown()
-    initialize_quality_dropdown()
+    clear_form()
```

Comment:
Made it mainly due to the call `initialize_slot_dropdown`, because on loading this function is not called and the incorrect point is established for `slot_dropdown`